ra/ra.go

```diff
@@ -385,6 +386,9 @@ func validateEmail(address string) error {
 			"invalid contact domain. Contact emails @%s are forbidden",
 			domain)
 	}
+	if _, err := iana.ExtractSuffix(domain); err != nil {
+		return berrors.InvalidEmailError("invalid contact domain, domain does not end in a IANA suffix")
```

Comment:
```suggestion
		return berrors.InvalidEmailError("email domain name does not end in a IANA suffix")
```

Fixing a comma splice, and removing the "invalid" which is implicit in the error type.